packages/language-server/src/plugins/html/features/astro-attributes.ts

```diff
@@ -75,6 +81,7 @@ export const astroDirectives = newHTMLDataProvider('astro-directives', {
 		{
 			name: 'client:load',
 			description: 'Start importing the component JS at page load. Hydrate the component when import completes.',
+			valueSet: 'v',
```

Comment:
So, I'm not sure why it's just named `v` instead of something descriptive but this changes the text edit for the completions to just be the value. 

ex: The `client:load` completions now create the following text: `client:load` instead of `client:load=""`